temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Using `pdb` (Python Debugger) should be avoided in production code. If this is for debugging purposes, it should be removed before committing the code.
```
Remove the line `import pdb`.
```